src/screens/VisualTests/VisualTests.stories.tsx

```diff
@@ -412,164 +461,190 @@ export const BrowserAddedInSelectedBuild: Story = {
-    }),
+    selectedBuildInfo: { buildId: pendingBuild.id, storyId: meta.args.storyId },
+    $graphql: {
+      AddonVisualTestsBuild: {
+        lastBuildOnBranch: withTests(pendingBuild, pendingTests),
+      },
+    },
   },
 } satisfies Story;
 
 /**
  * As above but we started the next build
  */
 export const PendingLocalBuildCapturing = {
-  parameters: {
-    ...withBuilds({
-      selectedBuild: withTests(pendingBuild, pendingTests),
-      lastBuildOnBranch: withTests({ ...startedBuild, id: "2" }, inProgressTests),
-    }),
-  },
   args: {
     ...EmptyBranchLocalBuildCapturing.args,
+    selectedBuildInfo: { buildId: pendingBuild.id, storyId: meta.args.storyId },
+    $graphql: {
+      AddonVisualTestsBuild: {
+        lastBuildOnBranch: withTests({ ...startedBuild, id: "2" }, inProgressTests),
+        selectedBuild: withTests(pendingBuild, pendingTests),
+      },
+    },
   },
 } satisfies Story;
 
 /**
  * The next build is snapshotting and has captured this story
  */
 export const PendingLocalBuildCapturedStory = {
-  ...PendingLocalBuildCapturing,
+  args: {
+    ...EmptyBranchLocalBuildCapturing.args,
+    selectedBuildInfo: { buildId: pendingBuild.id, storyId: meta.args.storyId },
+    $graphql: {
+      AddonVisualTestsBuild: {
+        lastBuildOnBranch: withTests({ ...startedBuild, id: "2" }, pendingTests),
+        selectedBuild: withTests(pendingBuild, pendingTests),
+      },
+    },
+  },
   parameters: {
-    ...withBuilds({
-      selectedBuild: withTests(pendingBuild, pendingTests),
-      lastBuildOnBranch: withTests({ ...startedBuild, id: "2" }, pendingTests),
-    }),
     ...withFigmaDesign(
       "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=2303-374529&t=qjmuGHxoALrVuhvX-0"
     ),
   },
+  // NOTE: this does not current work, the story is auto-selected
+  // play: async ({ canvasElement, args, argsByTarget }) => {
+  //   // We have to wait a moment for the story to render
+  //   const canvas = within(canvasElement);
+  //   await canvas.findAllByText("Latest");
+
+  //   // Ensure we don't switch to the new build, the user has to opt-in
+  //   mock(args.setSelectedBuildInfo!).mock.calls.forEach(([updater]) => {
+  //     const result = typeof updater === "function" ? updater(args.selectedBuildInfo) : updater;
+  //     expect(result).toEqual(args.selectedBuildInfo); // Unchanged
+  //   });
+
+  //   // Now opt in
+  //   const viewLatestSnapshot = (await canvas.findAllByText("View latest snapshot"))[0];
+  //   await userEvent.click(viewLatestSnapshot);
+
+  //   const graphqlArgs = argsByTarget.graphql?.$graphql as typeof args.$graphql; // We need to type argsByTarget
+  //   await waitFor(() => {
+  //     expect(args.setSelectedBuildInfo).toHaveBeenCalledWith({
+  //       buildId: graphqlArgs?.AddonVisualTestsBuild?.lastBuildOnBranch?.id,
+  //       storyId: meta.args.storyId,
+  //     });
+  //   });
+  // },
```

Comment:
As noted this play function is disabled here (and in 2 other tests) but fixed in https://github.com/chromaui/addon-visual-tests/pull/130